integration/util.py

```diff
@@ -80,6 +79,8 @@ class MagicFolderEnabledNode(object):
     temp_dir = attr.ib()
     name = attr.ib()
 
+    action = attr.ib()
```

Comment:
This could use some docs.  It took me a couple tries to understand the `request.addfinalizer(action.finish)` bits below but now I see it is related to this attribute.

Reply:
Not exactly, but close enough. I've added some docs for this attribute and that call.